src/main/java/org/repodriller/scm/GitRepository.java

```diff
@@ -455,23 +467,39 @@ public String getCommitFromTag(String tag) {
 			Repository repo = git.getRepository();
 
 			Iterable<RevCommit> commits = git.log().add(getActualRefObjectId(repo.findRef(tag), repo)).call();
-			
+
 			for(RevCommit commit : commits) {
 				return commit.getName().toString();
 			}
-			
+
 			throw new RuntimeException("Failed for tag " + tag); // we never arrive here, hopefully
 
 		} catch (Exception e) {
 			throw new RuntimeException("Failed for tag " + tag, e);
 		}
 	}
-	
+
 	private ObjectId getActualRefObjectId(Ref ref, Repository repo) {
 		final Ref repoPeeled = repo.peel(ref);
 		if(repoPeeled.getPeeledObjectId() != null) {
 			return repoPeeled.getPeeledObjectId();
 		}
 		return ref.getObjectId();
 	}
+
+	public String getPath() {
```

Comment:
no need for a getter and setter

Reply:
Setter is needed so that the empty constructor works. I've removed the getter. See a03dbd8.